leonardo-example.conf

```diff
@@ -9,6 +9,23 @@ dataproc {
   leoGoogleProject = "GOOGLE_PROJECT"    # the name of the google project to use during cluster startup
                                          # *this is not the project the cluster will be created in
   clusterUrlBase = "https://YOUR_DOMAIN/notebooks" # the base url to access your cluster
+
```

Comment:
I moved `networkTag`, `firewallRuleName`, `vpcNetwork`, and `vpcSubnet` from `ProxyConfig` to `DataprocConfig`. They are used in Dataproc cluster setup, so didn't really make sense in `ProxyConfig`.